hal_st/stm32fxxx/TimerStm.cpp

```diff
@@ -46,14 +48,20 @@ namespace
 #if defined(TIM6)
 #if defined(STM32F7) || defined(STM32F4)
         IRQn_Type::TIM6_DAC_IRQn,
+#elif defined(STM32G071xx)
```

Comment:
Is this specific for the G071, or generic for all G0xx?

Reply:
See my other reply